packages/angular/projects/ui-angular/src/lib/services/__tests__/authenticator.service.test.ts

```diff
@@ -10,7 +10,7 @@ export class MockAuthService {
 
   subscribe(callback: () => void) {
     this.listeners.push(callback);
-    const unsubscribe = () => {};
+    const unsubscribe = () => jest.fn();
```

Comment:
_Should_ `unsubscribe` return another function? Seems like this could just be:
```
const unsubscribe =  jest.fn();
```

But maybe I'm missing something?